tests/Oryx.Tests.Common/TestConstants.cs

```diff
@@ -12,5 +14,8 @@ public static class TestConstants
         // Xunit trait values
         public const string Category = "Category";
         public const string Release = "Release";
+
+        //AI key 
+        public const string AiKey = "InstrumentationKey=test";
```

Comment:
nit: I'd rename this to `AppInsightsConnectionString` so we're more clear about what this string is